### ISSUE TITLE
no response from fresh gpttools installation

### ISSUE THREAD

**Issue by stranda** [JamesHWade/gpttools#53]
Thanks for this software and I'm looking forward to using it during development.  However, I'm having trouble getting any of the features to work.

I've installed gpttools (using install_github()) and had no problems noted in the install.  

The OPENAI_API_KEY is set and works with gptstudio without a problem.

the gpttools features are included in the addins menu and are available through Ctrl+Shift+P

However, when I try to use any of the gpttools functions (roxygen documentation, script-to-function, etc) nothing is happening. 

By nothing I mean there are no messages in the console or in the background-jobs pane.  Furthermore, there is no evidence of increased cpu usage by rsession or rstudio.

I've tried on several installs on different linux boxes.  
I'm seeing this behavior on Rstudio desktop version 2023.06.1 Build 524  and both R 4.3.0 and R 4.3.1.

Given that there don't seem to be other similar issues reported, I'm assuming that I am doing something wrong that is causing the behavior.   I'd appreciate any pointers you may have to guide me back on track

thanks 
allan

**Comment by JamesHWade** [comment]
Thank you for reporting the issue. I'll look into it.

**Comment by JerePlum99** [comment]
@JamesHWade Appreciate the package and loved your presentation at conf - I've installed and am running into a similar issue. 

My pak::pak() install failed to actually add any fo the functions to the `Addins` tab, I reinstalled with the other method and it added the functionality to `Addins`. Unfortunately, when I run the options over existing code it only shows the request and results in the console, and fails to actually replace or add in code to the open R files. I have `gptstudio` loaded as well and all that functionality is working fine. 

Thanks again and let me know if I can help in any way!
